docs/web-wallet.md

```diff
@@ -0,0 +1,249 @@
+
+:::
+
+:::note
+
+In Case of Off-Chain Verification, once the user goes through the flow of verification in the Web Wallet, the zero-knowledge proof is returned as a JWZ token in a callback to your verifier backend. You need to handle this callback coming from the Web Wallet and verify the proof.
+
+:::
+
+### Step 4 - Integrating the Web Wallet on the Client Side
+
+Integrating the tool on the client side is straightforward. Simply redirect users from your application to the Web Wallet URL. This URL must include the verification request encoded in base64 in the fragment of the URL.
+
+<Tabs>
+<TabItem value="Polygon Amoy">
+
+```bash
+https://web-wallet-test.polygonid.me/#base64EncodedData
+```
+
+</TabItem>
+
+<TabItem value="Polygon Main">
+
+```bash
+https://web-wallet.polygonid.me/#base64EncodedData
+```
+
+</TabItem>
+</Tabs>
+
+:::caution
+
+For security reasons, integration of this tool via IFrame is not supported. Redirecting users is the recommended method for accessing our Web Wallet to ensure secure interaction.
+
+:::
+
+Below is a code example for integrating the Web Wallet on the client side in JS:
+<Tabs>
+<TabItem value="Off-Chain Verification">
+
+```js
+// Define the verification request
+const verificationRequest = {
+  backUrl: "https://my-app.org/back",
+  finishUrl: "https://my-app.org/finish",
+  logoUrl: "https://my-app.org/logo.png",
+  name: "My app",
+  zkQueries: [
+    {
+      circuitId: "credentialAtomicQuerySigV2",
+      id: 1711399135,
+      query: {
+        allowedIssuers: ["*"],
+        context:
+          "https://raw.githubusercontent.com/anima-protocol/claims-polygonid/main/schemas/json-ld/pol-v1.json-ld",
+        type: "AnimaProofOfLife",
+        credentialSubject: {
+          human: {
+            $eq: true,
+          },
+        },
+      },
+    },
+  ],
+  callbackUrl: "https://my-app.org/api/callback",
+  verifierDid: "did:polygonid:polygon:amoy:2qV9QXdhXXmN5sKjN1YueMjxgRbnJcEGK2kGpvk3cq",
+};
+
+// Encode the verification request to base64
+const base64EncodedVerificationRequest = btoa(JSON.stringify(verificationRequest));
+
+// Open the Polygon ID Verification Web Wallet with the encoded verification request
+window.open(`https://web-wallet.polygonid.me${base64EncodedVerificationRequest}`);
+```
+
+</TabItem>
+
+ <TabItem value="On-Chain Verification">
+
+```js
+// Define the verification request
+const verificationRequest = {
+  backUrl: "https://my-app.org/back",
+  finishUrl: "https://my-app.org/finish",
+  logoUrl: "https://my-app.org/logo.png",
+  name: "My app",
+  zkQueries: [
+    {
+      circuitId: "credentialAtomicQuerySigV2",
+      id: 1711399135,
+      query: {
+        allowedIssuers: ["*"],
```

Comment:
We need to add a warning/alert box clarifying that this value needs to be populated with the DIDs of actual issuers you trust.

Similar to what we do [here](http://localhost:3000/docs/verifier/verification-library/zk-query-language/#multi-query) and [here](http://localhost:3000/docs/verifier/verification-library/verifier-setup/#verifier-server-setup).

Reply:
added